cpp/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ cmake_minimum_required (VERSION 3.1)
 
 project (libphonenumber)
 set (libphonenumber_VERSION_MAJOR 7)
-set (libphonenumber_VERSION_MINOR 0)
+set (libphonenumber_VERSION_MINOR 2)
```

Comment:
This doesn't seem to have been changed automatically. It was in https://github.com/googlei18n/libphonenumber/pull/1031/commits/96051cb910f3e2e598b8c705767e7ad9a500d2d6

Do you need the manual changes? Otherwise you can remove them from your branch before sending a pull request. @dlyongemallo can explain how to do this.